{{ cookiecutter.package_name }}/setup.cfg

```diff
@@ -21,7 +21,7 @@ addopts = -p no:warnings
 [ah_bootstrap]
 auto_use = True
 
-[pycodestyle]
+[flake8]
```

Comment:
while you're at it, could you please bring over more error code, too? There is a more complete list in https://github.com/astropy/astropy/pull/7727
(at least I think that the template should have the most enabled, and then packages can disable by their liking as usually it's not happening the other way around)

Reply:
Oh, I've given up on "edit on github" a long time ago since it messed up things so many times.

Reply:
It is about 50-50 hit or miss for me... :crossed_fingers: